Makefile

```
@@ -21,17 +21,17 @@ LDFLAGS = -lpthread -lncursesw
 
 all: $(OBJECTS) $(SERVER_TARGET) $(CLIENT_TARGET)
 
-$(SERVER_TARGET): $(addprefix $(BUILD_DIR)/, server.o blokus.o sock.o)
-	@$(CC) -o $@ $^ $(LDFLAGS)
+$(SERVER_TARGET): $(addprefix $(BUILD_DIR)/, server.o blokus.o)
```

`sock.o` seems missing again, and also, we may need to add `frame.o` so that we could use `get_frame` function
```suggestion
$(SERVER_TARGET): $(addprefix $(BUILD_DIR)/, server.o blokus.o sock.o frame.o)
```